code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader"
+
```

Comment:
You did the loader, cool! 🥳 Also you remembered that space between external and internal imports (I am not sure if I use the correct terms) in every file and all your files are clean and well structured.